composer.json

```diff
@@ -55,7 +55,7 @@
     "extra": {
         "typo3/cms": {
             "cms-package-dir": "{$vendor-dir}/typo3/cms",
-            "extension-key": "skilldisplay",
+            "extension-key": "skilldisplay-content",
```

Comment:
That key is not valid, cited from docs:
> 2 .The extkey MUST be made up of lowercase alphanumeric characters and underscores only and MUST start with a letter.
>  —https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/NamingConventions/Index.html#extension-key-extkey

```suggestion
            "extension-key": "skilldisplay_content",
```

Reply:
args... totally right. no need to cite the docs, I was just "stupid". Will fix asap